whitepapers/0001_generic_message_passing.md

```diff
@@ -139,31 +133,31 @@ VAA struct {
 	// Tracked per (EmitterChain, EmitterAddress) tuple.
 	Sequence uint64 // <-- NEW
 
-    // Level of consistency requested by the emitter.
-    //
-    // The semantic meaning of this field is specific to the target
-    // chain (like a commitment level on Solana, number of
-    // confirmations on Ethereum, or no meaning with instant finality).
-    ConsistencyLevel uint8 // <-- NEW
+	// Level of consistency requested by the emitter.
+	//
+	// The semantic meaning of this field is specific to the target
```

Comment:
Is this a place to mention the "magic numbers" of 200 and 201, and how on EVM every other value means "finalized"?

Reply:
I was iffy about that since that's what it *currently* means but that wasn't the case for older messages. I will include an additional section explaining this though, because I think it's worth mentioning somewhere in this repo.